pytorch_lightning/loops/fit_loop.py

```diff
@@ -30,17 +33,27 @@ class FitLoop(Loop):
 
     Args:
         min_epochs: The minimum number of epochs
-        max_epochs: The maximum number of epochs
+        max_epochs: The maximum number of epochs, can be set -1 to turn this limit off
     """
 
-    def __init__(self, min_epochs: Optional[int] = None, max_epochs: Optional[int] = None):
+    def __init__(
+        self,
+        min_epochs: int = 0,
+        max_epochs: Optional[int] = None,
+        max_steps: Optional[int] = -1,
+        max_time: Optional[Union[str, timedelta, Dict[str, int]]] = None
+    ):
         super().__init__()
-        # Allow max_epochs or max_steps to be zero, since this will be handled by fit_loop.done
-        if max_epochs and max_epochs < -1:
+        if max_epochs < -1:
```

Comment:
this will raise an error if `max_epochs` is None. 
```suggestion
        if max_epochs is not None and max_epochs < -1:
```

Reply:
Or:

```python
if max_epochs is None:
    max_epochs = 1k
elif max_epochs < -1:
    fail() 
```

which is the same pattern we have for `max_steps` below